.travis.yml

```diff
@@ -14,7 +14,7 @@ jdk:
   - oraclejdk8
 
 script:
-  - ./gradlew clean check -x checkstyleTest --stacktrace
+  - ./gradlew clean check -x checkstyleTest -x :apollo-gradle-plugin:test --stacktrace
```

Comment:
Sorry but what do you mean.
Because of we updated snapshot version, all plugin tests will be failed as they depends on published version of snapshot api to maven. This flag supposes to disable just running unit tests, but compilation is still enabled. So not sure why packaging should fail if the compilation succeed